rxnetty-http/src/main/java/io/reactivex/netty/protocol/http/client/internal/RawRequest.java

```diff
@@ -166,7 +175,7 @@ public Observable asObservable(Connection<?, ?> connection) {
         return toReturn;
     }
 
-    private HttpRequest _copyHeaders() {
+    public HttpRequest _copyHeaders() {
```

Comment:
No, it was really just for consistency with "public HttpRequest _copyHeaders(String uri, HttpMethod method)" -- they should probably either both be public or both be private, I'm fine either way.